comms/lighthouse/src/server.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
What does this do?

Reply:
This comments avoids `eslint` to fail when the annotation `@ts-ignore` is used